cowprotocol/cowswap
# fix(trade): display Price updated banner in Confirmation modal

# Summary

Fixes #2639

Before:
![image](https://github.com/cowprotocol/cowswap/assets/7122625/129b8ec7-f96e-4113-9df9-93213a865007)

Now:
<img width="400" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/fd465262-1ebd-48f2-9fb5-a7c64e021aa1">

<img width="400" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/0988e5b6-0ead-4dd0-b143-4a31f840d9de">

  # To Test (Swap/Limit/Advanced)

1. Open a confirmation order
2. Wait until price is changed
- [ ] The `Price updated` banner should be shown
- [ ] The confirmation button should be disabled
2. Click on "Accept"
- [ ] The `Price updated` banner should be hidden
- [ ] The confirmation button should be enabled
